diff -u -r -N a/SDK/simVis/CMakeLists.txt b/SDK/simVis/CMakeLists.txt
--- a/SDK/simVis/CMakeLists.txt	2018-03-28 13:31:42.657329896 -0400
+++ b/SDK/simVis/CMakeLists.txt	2018-03-28 13:47:17.916199620 -0400
@@ -305,7 +305,6 @@
 # Newer than 1.7.0 release adds clear() to Profiler
 check_osgearth_method_exists(HAVE_OSGEARTH_PROFILER_CLEAR "osgEarth/Profiler" "clear")
 if(HAVE_OSGEARTH_PROFILER_CLEAR)
-    set(SDK_OSGEARTH_DEVEL 1)
     # Next anticipated version is 1.8.0
     set(SDK_OSGEARTH_VERSION_MAJOR 1)
     set(SDK_OSGEARTH_VERSION_MINOR 8)
